index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Mobile Portfolio Setup</title>
+</head>
+
+<body>
+    <header class="header padding-left-right">
+        <h2 class="logo">John Doe</h2>
```

Comment:
### **Nice Job so far, you are doing great**

- [ ] Kindly add a link to the logo with an `a tag`, this is for three reasons: convention, semantics, and user experience(UX).
[Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

Reply:
Okay Thank you @CollinsTatang 😊